libs/datagrid/src/lib/cells/order-type-cell.tsx

```diff
@@ -23,7 +24,13 @@ export const OrderTypeCell = ({
     }
     if (!value) return '-';
     if (order?.peggedOrder) {
-      return t('Pegged');
+      const offset = addDecimalsFormatNumber(
+        order.peggedOrder?.offset,
+        order.market.decimalPlaces
+      );
+      const reference =
+        Schema.PeggedReferenceMapping[order.peggedOrder?.reference];
+      return t('%s - %s Peg limit', [reference, offset]);
```

Comment:
**question (non-blocking)**:  Is this going to cover all the possible pegs? Just worrying if it'll mis-describe some

Reply:
agreed, just updated! it needs to be different based on buy/sell